platform/android/java/app/src/com/godot/game/GodotApp.java

```diff
@@ -34,11 +34,20 @@
 
 import android.os.Bundle;
 
+import com.godot.game.BuildConfig;
+
 /**
  * Template activity for Godot Android builds.
  * Feel free to extend and modify this class for your custom logic.
  */
 public class GodotApp extends GodotActivity {
+	static {
+		// .NET libraries.
+		if (BuildConfig.FLAVOR.equals("mono")) {
+			System.loadLibrary("System.Security.Cryptography.Native.Android");
```

Comment:
Can you surround this with `try...catch`. This allows to catch any errors if for some reasons we're unable to load the library (e.g: https://github.com/GodotVR/godot_openxr_vendors/blob/master/godotopenxrmeta/src/main/java/org/godotengine/openxr/vendors/meta/GodotOpenXRMeta.kt#L61-L66).

Reply:
Done.